tests/ert3/console/integration/test_cli.py

```diff
@@ -68,6 +69,64 @@ def test_cli_init_subfolder(workspace):
             ert3.console._console._main()
 
 
+@pytest.mark.requires_ert_storage
+def test_cli_init_invalid_example(tmpdir, ert_storage):
+    with tmpdir.as_cwd():
+        args = ["ert3", "init", "--example", "something"]
+        with patch.object(sys, "argv", args):
+            with pytest.raises(
+                SystemExit, match="Example something is not a valid ert3 example."
+            ):
+                ert3.console.main()
+
+
+@pytest.mark.requires_ert_storage
+def test_cli_init_example(tmpdir, ert_storage):
+    with tmpdir.as_cwd():
+        args = ["ert3", "init", "--example", "polynomial"]
+        with patch.object(sys, "argv", args):
+            ert3.console.main()
+        polynomial_path = pathlib.Path(tmpdir / "polynomial")
+        if polynomial_path.exists() and polynomial_path.is_dir():
```

Comment:
Why is this test gracefully handing the case where `polynomial_path` does not exist? Should it not fail if it is not present? To me it seems like the `if` should be made into an `assert` and the following `assert` should be moved one indentation line in 🤔

Reply:
good point